sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -52,7 +52,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>1.7.0-beta.1</version> <!-- {x-version-update;beta_com.azure:azure-core-amqp;dependency} -->
+      <version>1.7.0-beta.2</version> <!-- {x-version-update;beta_com.azure:azure-core-amqp;dependency} -->
```

Comment:
One question, should servicebus use  the unreleased 1.7.0-beta.3 in master?

Reply:
Yes, because @hemanttanwar is working on Amqp message types that are not yet released and are referenced in Service Bus.